jekyll/_posts/2023-08-01-nim-v20-released.md

```diff
@@ -0,0 +1,369 @@
+---
+title: "Nim v2.0 released"
+author: The Nim Team
+---
+
+The Nim team is proud and happy to announce Nim version 2.0.
+
+This is an evolution (not revolution) of Nim, bringing ORC memory management as a default, along with many other new features and improvements.
+
+
+
+
+## Installing Nim 2.0
+
+### New users
+
+Check out if the package manager of your OS already ships version 2.0 or
+install it as described [here](https://nim-lang.org/install.html).
+
+
+### Existing users
+
+If you have installed a previous version of Nim using `choosenim`,
+getting Nim 2.0 is as easy as:
+
+```bash
+$ choosenim update stable
+```
+
+Alternatively, you can download Nim 2.0 from
+[our nightlies builds](https://github.com/nim-lang/nightlies/releases/tag/2023-08-01-version-2-0-a488067a4130f029000be4550a0fb1b39e0e9e7c).
+
+
+
+## Donating to Nim
+
+We would like to encourage you to donate to Nim.
+The donated money will be used to further improve Nim by creating bounties
+for the most important bugfixes and features.
+
+You can donate via:
+
+* [Open Collective](https://opencollective.com/nim)
+* Bitcoin: `1BXfuKM2uvoD6mbx4g5xM3eQhLzkCK77tJ`
+
+If you are a company, we also offer commercial support.
+
+
+
+## New features
+
+### Better tuple unpacking
+
+Tuple unpacking for variables is now treated as syntax sugar that directly
+expands into multiple assignments. Along with this, tuple unpacking for
+variables can now be nested.
+
+```
```

Comment:
```suggestion
```nim
```

Reply:
No, I removed highlighting on purpose, here and in two more places, because the highlighter doesn't like underscores and makes the whole code-block ugly.

Reply:
This doesn't render well and so the highlighter was disabled for this snippet.